ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/methods/engine/AbstractEngineNewPayload.java

```diff
@@ -104,8 +104,13 @@ public JsonRpcResponse syncResponse(final JsonRpcRequestContext requestContext)
     final Optional<List<Withdrawal>> maybeWithdrawals =
         Optional.ofNullable(blockParam.getWithdrawals())
             .map(ws -> ws.stream().map(WithdrawalParameter::toWithdrawal).collect(toList()));
-    if (!getWithdrawalsValidator(timestampSchedule, blockParam.getTimestamp())
-        .validateWithdrawals(maybeWithdrawals)) {
+    final Optional<BlockHeader> maybeParentHeader =
+        protocolContext.getBlockchain().getBlockHeader(blockParam.getParentHash());
+    boolean ableToValidateWithdrawals = maybeParentHeader.isPresent();
```

Comment:
This does change behaviour such that for an "invalid withdrawals"-block, we will end up returning SYNCING if the parent header is missing. Before this PR, we would return an "Invalid params" error, however that may be incorrect - I am waiting fir clarification about the spec w.r.t syncing.